lib/adapters/http/index.js

```diff
@@ -705,6 +705,15 @@ function HttpPouch(opts, callback) {
         encodeURIComponent(JSON.stringify(opts.startkey)));
     }
 
+    // If opts.start_key exists, add the start_key value to the list of
+    // parameters.
+    // If start_key is given then the returned list of documents will
+    // start with the document whose id is start_key.
+    if (opts.start_key) {
+      params.push('start_key=' +
+        encodeURIComponent(JSON.stringify(opts.start_key)));
+    }
+
     // If opts.endkey exists, add the endkey value to the list of parameters.
```

Comment:
So this will work but I think it would be better to avoid a little duplication of the logic here. If we did

   if (opts.start_key) { 
     opts.startkey = opts.start_key;
   }

Then it means we leave all the encoding / JSON work in one place and makes it clear/explicit that if you pass in `start_key` then all we (or couchdb) does is pretend you said `startkey`. I just realised I wrote those steps but didnt actually explain the bug properly, sorry.

Another note is we need to do the same logic for `end_key` => `endkey`